app/models/entity.rb

```diff
@@ -0,0 +1,4 @@
+class Entity < ApplicationRecord
```

Comment:
- [ ] I came across this issue! I was able to create a transaction with empty fields :smile: You are supposed to validate the user's input properly together with validation messages. Validation gives users a good experience. Let's kindly prevent what's on the screenshot below from happening.

![Screenshot from 2022-12-10 15-14-48](https://user-images.githubusercontent.com/63087491/206860175-19004180-adc6-4db3-8428-5eee700339b9.png)